docs/userguides/contracts.md

```diff
@@ -256,6 +256,24 @@ If you need the `return_value` from a transaction, you have to either treat tran
 assert receipt.return_value == 123
 ```
 
+Transactions may also fail.
+When a transaction fails, Ape (by default) raises the virtual machine error which may crash your script.
+You will see exception trace data like this:
+
+```
+  File "$HOME/ApeProjects/ape-playground/scripts/fail.py", line 7, in main
+    contract.setNumber(5, sender=owner)
+
+ERROR: (ContractLogicError) Transaction failed.
+```
+
```

Comment:
Is there a section somewhere that talks about decoding custom errors? this might be a good place for it.

Reply:
There is one in testing: https://docs.apeworx.io/ape/stable/userguides/testing.html#custom-errors
Perhaps we need a separate `reverts.md` guide and reference it from both spots.

Reply:
Yeah, it might be a great candidate for a separate user guide

Reply:
done